Helper/Quote.php

```diff
@@ -130,4 +93,22 @@ public function getIsQuoteMultiShippingWithMerchantReference(string $merchantRef
 
         return $quote->getIsMultiShipping();
     }
+
+    /**
+     * Try to disable a quote after successful payment
+     * @param $quoteId
+     */
+    public function disableQuote($quoteId)
```

Comment:
In both the catch and the if statement could you either log a message, or elso throw the exception so that it is handled somewhere else instead of failing silently?

Also can you return the quote so that we can easily unit test the function?

Reply:
The side effect of failing to disable a quote is that the cart will remain active. If that occurs often, bug reports can easily describe such symptom without logs. Throwing an exception will push the issue to merchants and it's a breaking change. 

This method performs a command, so ideally it should not return anything. An easy way to test is to spy the dependency interactions (e.g. with `quoteRepository` and `cartRepository`).

Reply:
Yea I agree that if we just threw the exception it will push the issue elsewhere, but if we then catch this exception wherever the `disableQuote` function is being called and log a message, the merchant should see no change from his side right?

Reply:
I merged `develop` into this branch and that broke GraphQL tests. Do you have any idea why @Morerice?